apps/web/src/pages/user/index.tsx

```diff
@@ -0,0 +1,60 @@
+import {useEffect, useState} from 'react'
+
+function user() {
+  let [data, setData] = useState()
+  let [isLoading, setLoading] = useState(false)
+  useEffect(() => {
+    setLoading(true)
+    fetch('http://localhost:3000/api/users')
+      .then(res => res.json())
+      .then(data => {
+        setData(data)
+        setLoading(false)
+      })
+  }, [])
+  let users: any = data
+  if (isLoading) return <p>Loading...</p>
+  if (!users) return <p>No profile data</p>
+  return (
+    <div>
+      {Object.entries(data).map(([key, value]) => {
```

Comment:
do not need to use this right? you only want to get `data.data` not iterate all of the keys